Services/CO.CDP.Organisation.WebApi/Model/OrganisationJoinRequest.cs

```diff
@@ -0,0 +1,11 @@
+namespace CO.CDP.Organisation.WebApi.Model;
+
+public record OrganisationJoinRequest
+{
+    public required Guid Id { get; init; }
+    public required Person Person { get; init; }
+    public required Organisation Organisation { get; init; }
+    public required Person ReviewedBy { get; init; }
+    public required DateTimeOffset ReviewedOn { get; set; }
+    public required string Status { get; set; }
```

Comment:
should it be enum?

Reply:
Yeah. I do actually have one.. Just forgot to use it here. I'll add it now.

Reply:
Done